engine/hexen2/Makefile

```diff
@@ -1015,7 +1017,7 @@ endif
 # for extra missing headers
 INCLUDES += -I$(OSLIBS)/amigaos/include
 # AmiTCP SDK
-NET_INC   = -I$(OSLIBS)/amigaos/netinclude
+#NET_INC   = -I$(OSLIBS)/amigaos/netinclude
```

Comment:
Why are we not using the RoadShow SDK?

Reply:
The original SDK had some compilation issues with some errno codes, so I opted to use the modified Roadshow SDK that comes with the GCC6 toolchain. I could probably work around this in a different way to keep the original SDK.

Reply:
I modified the Makefiles to use the BEBBO_TOOLCHAIN switch for this as well.